integration/tests/basic_functionality/install_pe_only_module_with_puppetfile.rb

```diff
@@ -0,0 +1,97 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'RK-158 - C92362 - Install a PE-only module from forge'
+
+#Init
+env_path = on(master, puppet('config print environmentpath')).stdout.rstrip
+r10k_fqp = get_r10k_fqp(master)
+master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
+
+git_repo_path = '/git_repos'
+git_repo_name = 'environments'
+git_control_remote = File.join(git_repo_path, "#{git_repo_name}.git")
+git_environments_path = '/root/environments'
+last_commit = git_last_commit(master, git_environments_path)
+git_provider = ENV['GIT_PROVIDER']
```

Comment:
You don't provide a default provider. Potentially could break if a user fails to provide one.